cg/constants/lims.py

```diff
@@ -60,6 +60,7 @@
         "Hybridize Library TWIST v1": "Bait Set",
         "Hybridize Library TWIST v2": "Bait Set",
         "Target enrichment TWIST v1": "Bait Set",
+        "Library Prep (Dev) v3": "Bait Set",
```

Comment:
Just a question on this as I am unaware of the proper usages. Seems like the master steps have quite distinc library capture kit namings. 

Will `Library Prep (Dev) v3` be unique enough in future?

Reply:
these names are forced unique in LIMS, so there wont be any future issues